src/components/pages/Announcements/Filter/index.tsx

```diff
@@ -119,7 +119,7 @@ const FilterAnnouncement = ({ closeModal, action }: TFilterAnnouncement) => {
             </div>
           </div>
           {fromDate && (
-            <div className="mt-4">
+            <div className={`mt-4 ${fromDate ? 'pb-4' : ''}`}>
```

Comment:
```suggestion
          <div className="my-4">
```
is there a reason why we can't use this?

Reply:
I have adjusted it